src/ops.rs

```diff
@@ -400,6 +400,7 @@ fn op_fetch(
   let cmd_id = base.cmd_id();
   let id = inner.id();
   let url = inner.url().unwrap();
+  let method = inner.method().unwrap_or("GET");
```

Comment:
I think `unwrap_or` is not needed, `method` is always passed in message.

Reply:
Will do that. Thanks!

Reply:
I think it's not yet resolved